app/commands/v2/put_content.rb

```diff
@@ -200,6 +202,29 @@ def clear_draft_item_of_different_locale_but_matching_base_path
         )
       end
 
+      def update_draft_items_of_different_locale_but_matching_content_id
+        return unless payload[:base_path]
+        return if Document.where(content_id: payload[:content_id], locale: payload[:locale]).any?
+
+        draft_editions_for_different_locale = Document
+          .where(content_id: payload[:content_id])
+          .where
+          .not(locale: payload[:locale])
+          .map(&:draft)
+          .compact
+
+        after_transaction_commit do
```

Comment:
What's the transaction here? Is it that the rest of the `call` method executes successfully and we only want to run this if it does? Or is it scoped to this method?

Reply:
Yes, it's after the `call` (I think it's [this line](https://github.com/alphagov/publishing-api/blob/5ad088fe246c3dcaa4f73efdb9e637df6b35eaed/app/commands/base_command.rb#L21) that executes the block), so we only send the payload to content store once the data has actually been committed to the database.